### PR TITLE
Updated Augments table.

### PR DESCRIPTION
**Pull request by IxAalexI**

### CODE REVIEW

File: sql/augments.sql [hunk 1]
```diff
@@ -112,9 +112,9 @@ INSERT INTO `augments` VALUES ('73', '0', '0', '0', '0'); -- 73 currently unused
 INSERT INTO `augments` VALUES ('74', '0', '0', '0', '0'); -- Cap. Point +1%
 INSERT INTO `augments` VALUES ('75', '0', '0', '0', '0'); -- Cap. Point +33%
 
--- 76 to 95 currently unused. Leave at zero.
-INSERT INTO `augments` VALUES ('76', '0', '0', '0', '0');
-INSERT INTO `augments` VALUES ('77', '0', '0', '0', '0');
+-- 78 to 95 currently unused. Leave at zero.
```
Comment (TeoTwawki) [inline sql/augments.sql]: move the comment to below new additions that use 77 and 77.

Comment (TeoTwawki) [inline sql/augments.sql]: likewise for any similar changes

**Comment by TeoTwawki** [comment]
I think that was the only one actually. Good job :+1:

**Comment by IxAalexI** [comment]
Oops, fixed.

**Pull request merged**